CHANGELOG.rst

```diff
@@ -6,6 +6,32 @@
 Synapse Changelog
 *****************
 
+v2.208.0 - 2025-04-17
+=====================
+
+Features and Enhancements
+-------------------------
+- Updated Cortex to run Storm package ``onload()`` handlers every time it
+  becomes active.
```

Comment:
suggestion to add something like "on boot or promotion to a leader" to clarify "leader"